plopfile.js

```diff
@@ -0,0 +1,5 @@
+const Component = require('./plop/component');
```

Comment:
kill plop if not needed here

Reply:
Remove, but not pushed yet - waiting for Sahar